engineering/documents/README.tmpl.md

```diff
@@ -0,0 +1,53 @@
+```
+logo if any exists
+```
+
+```
+row of badgesyou might consider necessary.
+Examples include:
+
+- godoc reference
+- build status
+- code coverage
+- go report
+- stability of the project
+```
+
+# Project Name
+
+```
+Short description of the project.
+Should mention the kind of project this is (library, demo, tool, etc.) and
+the current [stability status](https://www.npmjs.com/package/stability-badges).
+```
+
+# Installation
+
+```
+Short installation guide or link to longer installation instructions.
+This should include a pre-requisites subsection if needed.
+
+Are there Docker images, packages managers (brew, apt, etc), installations scripts?
+```
+
+# Example of utilization
+
+```
```

Comment:
For some langauages we should define the style of the examples, for example in go, the example should be from a extnernal point of view (good), or for a internal point of view of the library (bad). Because some example doesn't include the package name for the symbols of the package on the example, because are code from a example in the package, and then you can't copy paste it.

Reply:
added